dxr/query.py

```diff
@@ -102,8 +102,27 @@ def results(self, offset=0, limit=100):
         # will OR the elements of the inner lists and then AND those OR balls
         # together.
```

Comment:
I didn't mean to pin it on you; I just noticed that it's in the wrong place while reading the code.